src/System.Net.Sockets.Legacy/System.Net.Sockets.Legacy.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
Also, we're still working on naming this one. (Although APM is a legacy/deprecated pattern.)